bios/stage-2/src/main.rs

```diff
@@ -27,6 +27,7 @@ const BOOTLOADER_SECOND_STAGE_PARTITION_TYPE: u8 = 0x20;
 const STAGE_3_DST: *mut u8 = 0x0010_0000 as *mut u8; // 1MiB (typically 14MiB accessible here)
 const STAGE_4_DST: *mut u8 = 0x0020_0000 as *mut u8; // 2MiB (typically still 13MiB accessible here)
 const KERNEL_DST: *mut u8 = 0x0100_0000 as *mut u8; // 16MiB
+const RAMDISK_DST: *mut u8 = 0x0400_0000 as *mut u8; // 64MiB
```

Comment:
Hardcoding an address here could be problematic because it effectively limits the possible kernel size.